src/components/NcRichText/NcReferenceList.vue

```diff
@@ -32,7 +32,7 @@ export default {
 			default: '',
 		},
 		referenceData: {
-			type: Object,
+			type: Array,
 			default: null,
```

Comment:
How can it be null but type `Array`? (null is object)
Should the default not be rather `() => []` (empty array?) or `undefined`?